contracts/interfaces/modules/licensing/ILicenseTemplate.sol

```diff
@@ -0,0 +1,133 @@
+/// A License Template is responsible for defining a template of license terms that allow users to create
+/// licenses based on the template terms.
+/// The License Template contract is also responsible for registering, storing, verifying,
+/// and displaying license terms registered with the License Template.
+/// Anyone can implement a License Template and register it into the Story Protocol.
+/// @dev The License Template should assign an unique ID to each license terms registered.
+interface ILicenseTemplate is IERC165 {
+    /// @notice Emitted when a new license terms is registered.
+    /// @param licenseTermsId The ID of the license terms.
+    /// @param licenseTemplate The address of the license template.
+    /// @param licenseTerms The data of the license.
+    event LicenseTermsRegistered(uint256 indexed licenseTermsId, address indexed licenseTemplate, bytes licenseTerms);
+
+    /// @notice Returns the name of the license template.
+    /// @return The name of the license template.
+    function name() external view returns (string memory);
+
+    /// @notice Converts the license terms to a JSON string.
+    /// @param licenseTermsId The ID of the license terms.
+    /// @return The JSON string of the license terms.
+    function toJson(uint256 licenseTermsId) external view returns (string memory);
+
+    /// @notice Returns the metadata URI of the license template.
+    /// @return The metadata URI of the license template.
+    function getMetadataURI() external view returns (string memory);
+
+    /// @notice Returns the total number of registered license terms.
+    /// @return The total number of registered license terms.
+    function totalRegisteredLicenseTerms() external view returns (uint256);
+
+    /// @notice Checks if a license terms exists.
+    /// @param licenseTermsId The ID of the license terms.
+    /// @return True if the license terms exists, false otherwise.
+    function exists(uint256 licenseTermsId) external view returns (bool);
+
+    /// @notice Checks if a license terms is transferable.
+    /// @param licenseTermsId The ID of the license terms.
+    /// @return True if the license terms is transferable, false otherwise.
+    function isTransferable(uint256 licenseTermsId) external view returns (bool);
+
+    /// @notice Returns the earliest expiration time among the given license terms.
+    /// @param start The start time.
+    /// @param licenseTermsIds The IDs of the license terms.
+    /// @return The earliest expiration time.
+    function getEarlierExpireTime(uint256 start, uint256[] calldata licenseTermsIds) external view returns (uint);
+
+    /// @notice Returns the expiration time of a license terms.
+    /// @param start The start time.
+    /// @param licenseTermsId The ID of the license terms.
+    /// @return The expiration time.
+    function getExpireTime(uint256 start, uint256 licenseTermsId) external view returns (uint);
```

Comment:
put licenseTermsId as the first argument, so that it is consistent with other functions.

(uint256 start, uint256 licenseTermsId) => (uint256 licenseTermsId, uint256 start)

Reply:
updated.